src/profile-logic/function-info.js

```diff
@@ -42,35 +42,52 @@ export function stripFunctionArguments(functionCall: string): string {
 }
 
 export function removeTemplateInformation(functionName: string) {
+  // The result for this function. This works as an accumulator.
   let result = '';
+  // This keeps the depth in the tree of anchorness.
   let depth = 0;
-  let start = 0; // Start of a segment we'd like to keep
+  // This keeps the depth start of a template we'd like to remove. This is
+  // useful to find the end of this template. This is null if we're not removing
+  // any template yet.
+  let templateStartDepth: null | number = null;
+  // Start of a segment we'd like to keep
+  let start = 0;
+
   // We start the loop at i = 1 because we don't want to extract any
   // template-like information starting at the beginning of the string:
   // templates can't occur before the name of a function, so this is certainly
   // an HTML tag like <script>.
   for (let i = 1; i < functionName.length; i++) {
-    // We also don't want to extract template-like information that start after
-    // a space, as that won't likely be a real template information and
-    // probably rather an HTML tag name.
-    if (functionName[i] === '<' && functionName[i - 1] !== ' ') {
-      if (depth === 0) {
-        // Template information begins, save segment
-        result += functionName.substr(start, i - start);
```

Comment:
note: I changed all `substr` to `slice`. In this specific line, this is a bit simpler (we just use `i` instead of computing the length), in the other line this makes no difference.